.gitignore

```diff
@@ -19,6 +19,8 @@
 /third_party/zlib/
 /third_party/swiftshader/
 /third_party/depot_tools/
+/third_party/libcurl/
+/third_party/expat/src/
```

Comment:
I think Kudo has added expat already in upstream .SO when we sync that chromium PR , we might get conflict. Better to keep only libcurl for now ?

Reply:
Agree. Will update the changes keeping only libcurl path.